src/librbd/io/ImageRequestWQ.h

```diff
@@ -75,6 +75,7 @@ class ImageRequestWQ
 
   void apply_qos_limit(uint64_t limit, const uint64_t flag);
 
+  void apply_qos_limit(uint64_t limit, uint64_t bursting, const uint64_t flag);
```

Comment:
Looks don't need a new define. just add a parameter into apply_qos_limit().

Reply:
:+1:

Reply:
@yangdongsheng My bad. Define a new one is to avoid changing the calling in test case.
How about moving the `flag` to the first argument ? 
such as:
`void apply_qos_limit(const uint64_t flag, uint64_t limit, uint64_t bursting);`

This looks clear when calling it.
Thanks.

Reply:
I am not sure I understand all what you want to say, but no need for a new function. And you need to update the unit-test at the same time, rather than avoid to change it.